site/docs/plugins/keyboard.md

```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
 
 Naturally, all other methods that send messages other than text messages support the same options, as specified by the [Telegram Bot API Reference](https://core.telegram.org/bots/api).
 
-If you want to specify further options with your message, you may need to create your own `reply_markup` object.
-In that case, you have to use `keyboard.build()` when passing it to your object.
+You can also give your keyboard one or more further properties by calling special methods on it.
+They will not add any buttons, but rather define the behavior of the keyboard:
```

Comment:
`.` instead of `:`?

Reply:
Both are OK to me here, I didn't suggest it because it was a long sentence. @KnorpelSenf makes final decision.

Reply:
Go ahead

Reply:
[bb048cd](https://github.com/grammyjs/website/pull/451/commits/bb048cdbd65b979de7bd6c43357166fbea19f426)